docs/EN_US/ECLWatch/ECLWa_mods/SessFaq.xml

```diff
@@ -0,0 +1,269 @@
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Does Auto Refresh of
+        active workunits and graphs extend your ESP session?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No, only user actions such
+        as typing or mouse clicks extend a session. </para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I have to login to
+        ECLWatch? </para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, just as you currently
+        do. </para>
+      </listitem>
+ 
+
+          <listitem>
+	<para><emphasis role="strong">Q</emphasis>: Will I have to login to the
+    ECL IDE?</para>
+
+    <para><emphasis role="strong">A</emphasis>: Yes, but you already should
+    be. There will be no apparent changes here only behind the scenes where
+    you are being authenticated. </para>
+     </listitem>
+
+	
+      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Will I have to login to
+    ConfigManager?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No. (out of scope)</para>
+     </listitem>
+
+	
+      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: What credentials should I use
+    to login with?</para>
+
+    <para><emphasis role="strong">A</emphasis>: (TBD) Use your Risk
+    credentials.</para>
+     </listitem>
+
+	
+	<listitem>
+    <para><emphasis role="strong">Q</emphasis>: Can I now be logged
+    off?</para>
+
+    <para><emphasis role="strong">A</emphasis>: Yes, you will be able to log
+    off, and if you do not your account will get locked after a configurable
+    period of inactivity.</para>
+     </listitem>
+
+	
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Will my sessions get logged
+    off due to inactivity?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No. After a configurable
+    period of inactivity your session will be locked. You will then need to
+    unlock to resume your session.</para>
+     </listitem>
+
+	
+	   <listitem><?dbfo keep-together="always"?>
+    <para><emphasis role="strong">Q</emphasis>: How long until my password
+    expires?</para>
+
+    <para><emphasis role="strong">A</emphasis>: This will depend on your
+    system policies. If your password does expire, you will be presented a
+    dialog to reset it.</para>
+     </listitem>
+
+	
+	<listitem>
+    <para><emphasis role="strong">Q</emphasis>: Can I log in concurrently with
+    different credentials?</para>
+
+    <para><emphasis role="strong">A</emphasis>: Yes, using different tabs in a
+    single browser, multiple instances of the same browser, or multiple
+    instances of different browsers.</para>
+     </listitem>
+
+	
+	<listitem>
+    <para><emphasis role="strong">Q</emphasis>: Will I still have to log into
+    ECLWatch in incognito mode, or delete the browser cookie cache, in order
+    to log in as a different user?</para>
```

Comment:
no need to discuss previous hacks/workarounds. The previous question already said you can log in using different credentials in different tabs.  The two answers can be combined.